library/include/G4CMPBoundaryUtils.hh

```diff
@@ -95,7 +115,7 @@ protected:
   G4double GetMaterialProperty(const G4String& key) const;
 
 private:
-  G4int buVerboseLevel;			// For local use; name avoids collisions
+  mutable G4int buVerboseLevel;			// For local use; name avoids collisions
```

Comment:
Why is this mutable?  SetVerboseLevel() is a non-const function (see above).  It should only be getting called from non-const process functions (like PostStepDoIt()).